src/editor/codemirror/language-server/signatureHelp.ts

```diff
@@ -67,9 +65,15 @@ export class SignatureHelpView
       this.triggerSignatureHelpRequest();
     } else if (docChanged) {
       const last = transactions[transactions.length - 1];
-      if (last.isUserEvent("input.type")) {
+      if (
+        last.isUserEvent("input.type") ||
+        last.isUserEvent("input.drop.apidocs")
```

Comment:
Something ought to explain this is used to communicate triggering signature help.